src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalForV2ApiTest.java

```diff
@@ -116,7 +116,7 @@ public void findOrgByPfmShouldBeSuccess() {
                 professionalApiClient.getMultipleAuthHeaders(pfmBearerToken));
         assertThat(response.get("paymentAccount")).asList().hasSize(3);
         assertThat(response.get("pendingPaymentAccount")).asList().hasSize(0);
-        assertThat(response.get("orgType")).isEqualTo("Doctor");
```

Comment:
why the orgType changed here?
can't we create new test case for the orgType "Solicitor"

Reply:
Opted to use one of the values from `OrganisationTypeConstants.java`

Reply:
fine then